apps/mobile/src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -217,7 +217,7 @@ const ListSpendables = ({
         />
       )}
       bounces={false}
-      contentContainerStyle={[a.px_lg]}
+      contentContainerStyle={{paddingHorizontal: 16}}
```

Comment:
is it the Style complaining about the types? my guess is they are not properly typing the Props.
```suggestion
      contentContainerStyle={{paddingHorizontal: 16}}
```

Reply:
Doesn't like the atoms inside an array, in this case since there's only 1 we can use it directly. Doing that now. I'll change them everywhere.